docs/frontend/development.md

```
@@ -51,6 +59,40 @@ frontend:
+
+<TabItem value="frontend outside Home Assistant">
+
+If you want to connect your development frontend to an existing home assistant instance without replacing the UI completely, you will need to add the url under which your development frontend is hosted in `configuration.yaml` of the home assistant it will be connecting to.
+
+```yaml
+http:
+  cors_allowed_origins:
+    - http://localhost:8124
+```
+
+After you've setup your frontend development environment so that you can run the `script/develop` script as described in section [Development](#development). You can use the following command as a replacement to develop and run the frontend on http://localhost:8124 and it will connect to the Home Assistant running on http://localhost:8123. Note that if you are running this command from a devcontainer, the url should be accessed from the container host.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)